packages/components/src/panel/README.md

```diff
@@ -1,71 +1,144 @@
 # Panel
+ Panels expand and collapse multiple sections of content.
+
+![](https://make.wordpress.org/design/files/2019/03/panel.png)
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+## Design guidelines
+
+### Anatomy
+
+A Panel consists of a single continuous column of subdivisions that contain items of content.
```

Comment:
Reading this leads me to believe the panel is the entire column, but it's actually just the individual expandable/collapsible section, right? I wonder if we could rephrase this to make that clear. Maybe something like:

> A Panel is a single section of content that can be expanded or collapsed as needed.